training_content/wiki_ed/slides/60-using-the-dashboard/6003-stage-1-bibliography.yml

```diff
@@ -0,0 +1,18 @@
+id: 6003
+title: "Stage 1: Bibliography"
+summary:
+content: |
+  <figure>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/9/96/Dashbooard.wikiedu.org_-_My_Articles_-_stage_1.png" />
+  </figure
+
+  You can see the details for each stage of the article development process by clicking bar
+  at the bottom of the article listing. For the first stage, compiling your set of sources,
+  use the **Bibliography** link. This will open a wiki sandbox page that includes some basic
+  structure to get you started.
+
+  Follow the trainings link to review the trainined modules related to this stage of the assignment.
+
+  Once you've compiled your initial bibliography, click **Mark Complete** to go to the next stage.
+  Keeping track of which step you're on will make it easier your instructor and your peer reviewers
```

Comment:
...will make it easier _for_ your instructor and your peer reviewers...

But, this is also not entirely true just yet.

Reply:
It's true that it *will*. It's also already true before we implement any Students tab changes, just because the sandboxes are clearly named so that if the instructor is reviewing the contribution history it will be obvious which one to look at.

Reply:
Nevermind. So yeah, not true yet. But it will be.